product-service/.serverless/cloudformation-template-create-stack.json

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
.serverless folder should be in the .gitignore file and must not be committed to repo

Reply:
agree .forgot to add to git ignore. but it's not a mistake